src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -58,22 +61,25 @@ private static VersionInfo GetVersionInfo()
                     loaderSignature: LoaderSignature);
             }
 
-            public TensorFlowMapper(IHostEnvironment env, ISchema inputSchema, byte[] modelBytes, string[] inputColNames, string outputColName)
+            public TensorFlowMapper(IHostEnvironment env, ISchema inputSchema, byte[] modelBytes, string[] inputColNames, string[] outputColNames)
             {
                 Contracts.CheckValue(env, nameof(env));
                 _host = env.Register("TensorFlowMapper");
                 _host.CheckValue(inputSchema, nameof(inputSchema));
                 _host.CheckNonEmpty(modelBytes, nameof(modelBytes));
                 _host.CheckNonEmpty(inputColNames, nameof(inputColNames));
-                _host.CheckNonEmpty(outputColName, nameof(outputColName));
+                _host.CheckNonEmpty(outputColNames, nameof(outputColNames));
 
                 _session = LoadTFSession(modelBytes, null);
-                _host.CheckValue(_session.Graph[outputColName], nameof(outputColName), "Output does not exist in the model");
                 _host.Check(inputColNames.All(name => _session.Graph[name] != null), "One of the input does not exist in the model");
+                _host.Check(outputColNames.All(name => _session.Graph[name] != null), "One of the output does not exist in the model");
```

Comment:
>[name] [](start = 69, length = 6)

Could this throw if "name" is null? If it does, then we need to add a check of each item in outputColNames, in addition to the CheckNonEmpty on the array itself.